packages/editor/src/components/post-last-revision/style.scss

```
@@ -1,15 +1,15 @@
 .editor-post-last-revision__title {
 	width: 100%;
-	font-weight: 600;
-
-	.dashicon {
-		margin-right: 5px;
-	}
+	font-weight: 500;
 }
 
-.components-button.editor-post-last-revision__title {
+.components-button.editor-post-last-revision__title.has-icon {
 	height: 100%;
 
+	&.has-icon {
+		justify-content: space-between;
+	}
```

```suggestion
.editor-post-last-revision__title.components-button.has-icon {
	height: 100%;
	justify-content: space-between;
```

Place the revision panel component-specific selector in front and remove the double `.has-icon` class.